src/components/Learn-Components/Pagination/index.js

```diff
@@ -50,13 +50,8 @@ const Pagination = ({ TOCData, chapterData, location, showQuizModal }) => {
           </div>
         </div>
       </PaginationWrapper>
-    ) : (
```

Comment:
@NikhilSharma03 could you please just comment it out, instead of removing it.

Reply:
Cause I see that the URL is not changing so it's gonna be hard to locate if we remove the button

Reply:
Yup, that would be better. Done.